packages/example/src/components/TokenList/TokenList.tsx

```diff
@@ -25,7 +25,15 @@ function useTokensBalance(tokenList?: any[], account?: string | null) {
 export function TokenList() {
   const { account, chainId } = useEthers()
   const { name, logoURI, tokens } = useTokenList(UNISWAP_DEFAULT_TOKEN_LIST_URI, chainId) || {}
-  const balances = useTokensBalance(tokens, account)
+  const { results: balances, error } = useTokensBalance(tokens, account)
+
+  if (error && error.event === 'noNetwork') {
+    return <span>Connect wallet to see tokens list</span>
+  }
+
```

Comment:
Btw is it expected behaviour that when I disconnect all of my wallets from the example website, the `chainId` is still equal `1`?